base/multi.jl

```diff
@@ -424,15 +436,44 @@ function get_bind_addr(w::Worker)
     get(w.config.bind_addr)
 end
 
+"""
+    myid()
+
+Get the id of the current process.
+"""
 myid() = LPROC.id
 
+"""
+    nprocs()
+
+Get the number of available processes.
+"""
 nprocs() = length(PGRP.workers)
+
+"""
+    nworkers()
+
+Get the number of available worker processes. This is one less than `nprocs()`. Equal to
+`nprocs()` if `nprocs() == 1`.
+"""
 function nworkers()
     n = nprocs()
     n == 1 ? 1 : n-1
 end
 
+"""
+    procs()
+
+Returns a list of all process identifiers.
+"""
 procs() = Int[x.id for x in PGRP.workers]
+
+"""
+    procs(pid::Integer)
+
+Returns a list of all process identifiers visible to worker `pid`.
+The result depends on the topology of the workgroup.
```

Comment:
While the statement above is not correct, it does seem more intuitive for `procs(pid)` to return workers based on topology.

I suggest we do not document this variant of `procs` for now. We can rename the current implementation of `procs(pid)` to an internal `procs_at_host(pid)` and change `procs(pid)` to return pids  based on topology.